addons/l10n_jo_edi/tests/test_edi_jo.py

```diff
@@ -0,0 +1,336 @@
+from odoo import Command
+from odoo.tests import tagged
+from odoo.tools import misc
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestEdiJo(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref='jo_standard'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.company = cls.setup_company_data('Jordan Company', chart_template=chart_template_ref)['company']
```

Comment:
No need. The whole setup already handle the creation of a sandboxed environment with a brand new company.